specification/marketplace/resource-manager/Microsoft.Marketplace/stable/2022-03-01/Marketplace.json

```
@@ -1925,12 +1925,12 @@
           "description": "Indicating whether all subscriptions are selected (=true) or not (=false).",
           "type": "boolean"
         },
-        "allItemsApproved": {
+        "approveAllItems": {
           "description": "Indicating whether all items are approved for this collection (=true) or not (=false).",
           "type": "boolean",
           "readOnly": true
         },
-        "allItemsApprovedModifiedAt": {
+        "approveAllItemsModifiedAt": {
           "description": "Gets the modified date of all items approved.",
```

Same here, we could add 
```suggestion
          "description": "Gets the modified date of all items approved.",
          "x-ms-client-name": "allItemsApprovedModifiedAt",
```